peekingduck/pipeline/nodes/preprocess/contrast.py

```diff
@@ -40,22 +36,26 @@ class Node(AbstractNode):
         |img|
 
     Configs:
-        brightness (:obj:`int`): **[-100,100], default = 0**. |br|
-            Adjusts the brightness of the image.
-        contrast (:obj:`float`): **[1, 3], default = 1**. |br|
-            Adjusts the contrast of the image.
+        alpha (:obj:`float`): **[1, 3], default = 1**. |br|
+            Increasing the value of alpha increases the contrast.
     """
 
     def __init__(self, config: Dict[str, Any] = None, **kwargs: Any) -> None:
         super().__init__(config, node_path=__name__, **kwargs)
 
+        if self.alpha > 3 or self.alpha < 1:
+            raise ValueError("alpha for contrast must be between [1, 3]")
+
     def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
-        """Draws pose details onto input image.
+        """Adjusts the contrast of an image frame.
 
         Args:
-            inputs (dict): Dictionary with keys "img".
+            inputs (Dict): Inputs dictionary with the key `img`.
+
+        Returns:
+            (Dict): Outputs dictionary with the key `img`.
         """
-        img = cv2.convertScaleAbs(
-            inputs["img"], alpha=self.contrast, beta=self.brightness
-        )
+
```

Comment:
Avoid whitespace between docstring and first line of code.

Reply:
Removed.